core/spec/models/spree/stock/estimator_spec.rb

```diff
@@ -140,6 +140,32 @@ module Stock
             expect(shipping_rates.first.tax_rate).to eq(tax_rate)
           end
         end
+
+        it 'uses the configured shipping rate selector' do
+          class Spree::Stock::TestSelector; end;
```

Comment:
There's always the opposite side : )

[On Mocks and Mockist Testing](http://jamesgolick.com/2010/3/10/on-mocks-and-mockist-testing.html)
https://vimeo.com/80533536